package.json

```diff
@@ -42,6 +42,7 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jest": "^27.9.0",
     "jest": "^29.7.0",
+    "socket.io-client": "^4.7.5",
```

Comment:
Do we really need the client dependency?

Reply:
We can remove it for now.